src/website/views/components/avatar.pug

```diff
@@ -156,27 +156,27 @@ ul#a-tabs--colors.a-tabs
         <i class="a-icon icon-user"></i>
       </div>
 
-h4 Inverse and transparent background
+h4 Light and transparent background
 p.-text
-  | Both icon and initial avatars support inverse version and transparent background.
+  | Both icon and initial avatars support light version and transparent background for dark backgrounds.
```

Comment:
@davidojedacl can you change from "light version and transparent background for dark backgrounds" to "light and transparent versions for dark backgrounds."?

Reply:
done